lib/tasks/create_staging_data.rake

```diff
@@ -172,15 +188,24 @@ staging = namespace :staging do
   task :sell_subscriptions => :environment do
     StagingHelper::switch_to_staging!
     percent = (ENV['PERCENT'] || '50').to_i / 100.0
-    sub_voucher = ValidVoucher.includes(:vouchertype).
-      where(:vouchertypes => {:category => :bundle, :subscription => true}).
+    sub_voucher1 = ValidVoucher.includes(:vouchertype).
+      where(:vouchertypes => {:category => :bundle, :subscription => true, :price => 70.00}).
+      first
+    sub_voucher2 = ValidVoucher.includes(:vouchertype).
+      where(:vouchertypes => {:category => :bundle, :subscription => true, :price => 80.00}).
       first
+    sub_voucher3 = ValidVoucher.includes(:vouchertype).
+      where(:vouchertypes => {:category => :bundle, :subscription => true, :price => 90.00}).
+      first
+    sub_vouchers = [sub_voucher1, sub_voucher2, sub_voucher3]
+
```

Comment:
same comment applies.  please fix this.  i cannot accept pull requests with copy-pasted code, this is a habit you should have kicked after cs61a!

Reply:
We're fixing this right now.

Reply:
Same here. Sorry for the repetition here, I have reverted reate_staging_data.rake back to its original version. I will open another feature branch for adding more subscription_vouchertypes.